### DISCUSSION TITLE
Keyboard heatmap

### DISCUSSION THREAD

**Discussion by nobe4** [monkeytypegame/monkeytype#4416] [Ideas] [answered]
Hi :wave: 

I am wondering it there's a plan/interest in having a per-letter heatmap that shows the differences between letters. 

Similarly to 
- [![image](https://github.com/monkeytypegame/monkeytype/assets/2452791/08d602f1-4042-45f5-8c46-eec0a8144bc2)](https://www.typesy.com/typesy-added-feature-keyboard-knowledge-heat-map-report/)
- [![image](https://github.com/monkeytypegame/monkeytype/assets/2452791/0104c858-1ee2-4e99-a81b-442a9d947303)](https://typemystuff.com/AppHelp/index.html#heatmap)

That could make for some great addition to the various graphs.
It would also show which letters need attention to be even more precise and fast :racehorse: 

Being a dev myself, I'd be happy to have a look at implementing this if deemed interesting :vulcan_salute: 

Best,

Edit: just found https://fingermap.monkeytype.com/, seems different but could help.

**Comment by Miodec** [comment, marked as answer]
The fingermap is just for sharing on how you actually type. Per key heatmap stats is definitely planned.

**Reply by nobe4** [reply]
Awesome, thanks for the answer, looking forward to that!
